ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/patch-panel.j2

```diff
@@ -0,0 +1,21 @@
+{% if patch_panel is arista.avd.defined %}
+!
+patch panel
+{%     for patch in patch_panel.patches | arista.avd.natural_sort %}
+   patch {{ patch.name }}
+{%         if patch.enabled is arista.avd.defined(false) %}
+      shutdown
+{%         endif %}
+{%         for connector in patch.connectors %}
```

Comment:
missing check for patch.connectors or you can add `| arista.avd.default([])` or you can use `| arista.avd.natural_sort` which also detects undefined gracefully.

Reply:
Fixed.